chaco/plots/api.py

```diff
@@ -0,0 +1,61 @@
+# (C) Copyright 2006-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+"""
+Defines the publicly accessible PlotRenderers in Chaco.
+
+- :class:`~.BarPlot`
+- :class:`~.CandlePlot`
+- :class:`~.CMapImagePlot`
+- :class:`~.ColorBar`
+- :class:`~.ColormappedScatterPlot`
+- :class:`~.ContourLinePlot`
+- :class:`~.ContourPolyPlot`
+- :class:`~.ErrorBarPlot`
+- :class:`~.FilledLinePlot`
+- :class:`~.HorizonPlot`
+- :class:`~.ImagePlot`
+- :class:`~.JitterPlot`
+- :class:`~.LineScatterPlot1D`
+- :class:`~.LinePlot`
+- :class:`~.MultiLinePlot`
+- :class:`~.PolarLineRenderer`
+- :class:`~.PolygonPlot`
+- :class:`~.QuiverPlot`
+- :func:`~.render_markers`
+- :class:`~.ScatterPlot`
+- :class:`~.ScatterPlot1D`
+- :class:`~.SegmentPlot`
+- :class:`~.TextPlot`
+- :class:`~.TextPlot1D`
+
+"""
+
+from chaco.plots.barplot import BarPlot
+from chaco.plots.candle_plot import CandlePlot
+from chaco.plots.cmap_image_plot import CMapImagePlot
+from chaco.plots.color_bar import ColorBar
+from chaco.plots.colormapped_scatterplot import ColormappedScatterPlot
```

Comment:
```suggestion
from chaco.plots.colormapped_scatterplot import ColormappedScatterPlot, ColormappedScatterPlotView
```

Reply:
can you also include this - and other missing imports i suggested in this module - to the module docstring?